src/tactic/ui/tools/pipeline_wdg.py

```diff
@@ -1,4 +1,4 @@
-###########################################################
+##########################################################
```

Comment:
don't make any change that's not related to this feature.  I suggest undoing that.